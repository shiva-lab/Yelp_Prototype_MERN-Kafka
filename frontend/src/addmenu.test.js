import React from 'react';
import { shallow } from 'enzyme';

import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });
import MyComponent from '../src/component/restaurantview/addmenu.js';
describe('MyComponent', () => {
    it('Rendering addmenu Page', () => {
        const component = shallow(<MyComponent debug />);
        expect(component).toMatchSnapshot();
    });
});