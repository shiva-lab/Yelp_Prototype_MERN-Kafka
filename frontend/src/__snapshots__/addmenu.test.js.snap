// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent Rendering addmenu Page 1`] = `ShallowWrapper {}`;
